dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/category/CategoryComboDeletionHandler.java

```diff
@@ -62,18 +67,28 @@ public String getClassName()
     {
         return CategoryCombo.class.getSimpleName();
     }
-    
+
     @Override
     public String allowDeleteCategory( Category category )
     {
```

Comment:
@vietnguyen Please, review this method in particular.
I changed from iterating through categoryOptionCombo to categoryCombo:

- the result should be the same
- in my opinion it makes more sense
- it makes the tests much easier